setup.py

```diff
@@ -67,6 +67,7 @@
         'rabbitmq': ['pika'],
         'clickhouse': ['clickhouse-driver'],
         'keycloak': ['python-keycloak'],
+        'arangodb': ['python-arango']
```

Comment:
```suggestion
        'arangodb': ['python-arango'],
```
Nit: this just makes the line changes a bit easier to read because the comma is already there when the next container is added.